packages/components/atoms/f-button/test/visual/f-button.visual.desktop.spec.js

```diff
@@ -141,4 +142,31 @@ describe('f-button Desktop visual tests', () => {
             });
         });
     });
+
+    describe('primary, secondary, inverse, ghostInverse, ghostTertiary, and ghost icon buttons in all 4 sizes', () => {
```

Comment:
Maybe a slightly more futureproof name
```suggestion
    describe('all valid iconButton types in all sizes', () => {
```